R/pathogen_summary.R

```diff
@@ -0,0 +1,92 @@
+#' Provides information on the delay distributions for a specific pathogen 
+#' stored in the `epiparameter` library.
+#'
+#' @inheritParams epidist
+#'
+#' @return An instance of a `epidist_tbl` class 
+#' @export
+#'
+#' @examples
+#' # summary of delay distributions available for ebola
+#' pathogen_summary(pathogen = "ebola")
+#' 
+#' # summary of delay distributions available for SARS CoV
+#' pathogen_summary(pathogen = "SARS_CoV")
+pathogen_summary <- function(pathogen) {
+  
+  pathogen <- match.arg(
+    arg = pathogen, 
+    choices = c(
+      "adenovirus", "ebola", "human_CoV", "influenza_A_seasonal", 
+      "influenza_B_seasonal", "influenza_H1N1p", "influenza_H7N9", "marburg", 
+      "measles", "MERS_CoV", "parainfluenza", "rhinovirus", "RSV", "SARS_CoV", 
+      "SARS_CoV_2_wildtype", "monkeypox"
+    ), 
```

Comment:
It seems like a lot of work to update this list each time new data is added to the spreadsheet. It might be a better strategy to generate this list dynamically.

For now, it's probably good enough to generate it each the function runs but in the future, we should probably figure out a way to generate it just once when the spreadsheet is updated.

Reply:
I agree, especially as the number of pathogens grows. I'll create an issue for this to be changed.